### ISSUE TITLE
[Feature request] Gamma correction on non-color maps

### ISSUE THREAD

**Issue by nate-brs** [AiSatan/node_wrangler_octane#16]
Hi, me again!
Going back to my first issue, I mixed up the gamma correction and the Float Image Tex.
There needs to be a change of gamma for non-color maps, their gamma needs to be set to 1.0 (defaults at 2.2). That includes all maps except diffuse/albedo and normal maps.
And minor correction on my previous issue: the bump map is imported as an Image Tex, it could benefit from using Float Image Tex like other non-color maps.

So to summarize:
- color maps (albedo, normal): Image Tex, gamma 2.2
- non-color maps (all the other maps): Float Image Tex, gamma 1.0

Thanks!

**Comment by AiSatan** [comment]
Hi!
I'll probably make it as a setting (def gamma: 1 or 2.2), 'cause nobody before mentioned this major thing, so, if someone like 2.2 for some reason, it won't be an issue for them. And for sure will change bump node.

FYI, I'm currently going to Europe, so, this issue will have to wait a 2-3 weeks before I comeback and change this, maybe you'll have something else meanwhile, be free to add here or create a new issue, thanks!
